google/cloud/storage/tests/curl_download_request_integration_test.cc

```diff
@@ -30,16 +30,16 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace internal {
 namespace {
 
+using ::google::cloud::internal::GetEnv;
 using ::google::cloud::testing_util::IsOk;
 using ::testing::Contains;
 using ::testing::HasSubstr;
 using ::testing::Pair;
 using ::testing::StartsWith;
 
-std::string HttpBinEndpoint() {
-  return google::cloud::internal::GetEnv("HTTPBIN_ENDPOINT")
-      .value_or("https://httpbin.org");
-}
+std::string HttpBinEndpoint() { return GetEnv("HTTPBIN_ENDPOINT").value(); }
+
+bool UsingEmulator() { return GetEnv("HTTPBIN_ENDPOINT").has_value(); }
```

Comment:
Opt Nit{sup N}: I might reverse the declaration order of these to somehow match the `has_value()` before `value()` ordering.

Reply:
Hmmm... Okay, done.